src/logic/Preferences.swift

```diff
@@ -28,6 +28,7 @@ class Preferences {
         "minDeminWindowShortcut": "M",
         "quitAppShortcut": "Q",
         "hideShowAppShortcut": "H",
+        "minimizeOtherWindowsShortcut": "1",
```

Comment:
I think adding the option to minimize other windows is not adding much noise in the existing dropdown "on release:". However, as a secondary shortcut like quit/hide/etc, I think it's too niche to have such a prominent UI in the preferences. This shortcut `1` would also conflict with #210. I think the feature is a bit too niche to have a dedicated shortcut as such.